README.md

```diff
@@ -175,4 +205,21 @@ To scan a GCP ...
 
 Using a computer already configured to use azure-cli, you may use Scout using the following command:
 
-    $ python Scout.py --provider azure
+    $ python Scout.py --provider azure --azure-cli
+    
+When using Scout in an Azure virtual machine with the Reader role on it's identity, you may use 
```

Comment:
Should be its instead of it's.

```suggestion
When using Scout in an Azure virtual machine with the Reader role on its identity, you may use 
```

We could also say the following though

```suggestion
When using Scout in an Azure virtual machine with the Reader role, you may use 
```